dds_web/utils.py

```diff
@@ -587,6 +587,53 @@ def calculate_version_period_usage(version):
     return bytehours
 
 
+def bytehours_in_last_30days(version):
+    """Calculate number of terrabyte hours stored in last 30 days."""
+    # Current date and date 30 days ago
+    flask.current_app.logger.debug("Project: %s \t Version: %s", version.project_id, version.id)
+    date_format = "%Y-%m-%d %H:%M:%S"  # No microseconds
+    now = datetime.datetime.strptime(current_time().strftime(date_format), date_format)
+    thirty_days_ago = now - datetime.timedelta(days=30)
```

Comment:
Since we have 28, 30 and 31 days in a month, using 30 days `timedeltta` will lead to missing some tbhours and double counting others. If I am not fooling myself, these errors will accumulate with time, won't they?
If so, then this should be a solution:
```suggestion
    month_ago = now - relativedelta(months=1)
```

Reply:
Ah. Yeah I didn't know relativedelta existed, but agreed, that's better if it works. I'll try it out 👍🏻